src/request_list.js

```diff
@@ -421,18 +423,18 @@ export class RequestList {
             throw new Error('The state object is invalid: nextIndex must be a non-negative number.');
         }
         if (state.nextIndex > this.requests.length) {
-            throw new Error('The state object is not consistent with RequestList: too few requests loaded.');
+            throw new Error('The state object is not consistent with RequestList too few requests loaded.');
         }
         if (state.nextIndex < this.requests.length
             && this.requests[state.nextIndex].uniqueKey !== state.nextUniqueKey) {
-            throw new Error('The state object is not consistent with RequestList: the order of URLs seems to have changed.');
+            throw new Error('The state object is not consistent with RequestList the order of URLs seems to have changed.');
         }
 
         const deleteFromInProgress = [];
         _.keys(state.inProgress).forEach((uniqueKey) => {
             const index = this.uniqueKeyToIndex[uniqueKey];
             if (typeof index !== 'number') {
-                throw new Error('The state object is not consistent with RequestList: unknown uniqueKey is present in the state.');
+                throw new Error('The state object is not consistent with RequestList unknown uniqueKey is present in the state.');
```

Comment:
I think the colon separated the two sentences. Maybe just replace it with a dot? `...RequestList. Unknown ...`